docs/reference/cat.asciidoc

```diff
@@ -115,7 +115,7 @@ case is the eighth one.
 
 [source,sh]
 --------------------------------------------------
-% curl '192.168.56.10:9200/_cat/indices?bytes=b' | sort -rnk8
+% curl '192.168.56.10:9200/_cat/indices?v&bytes=b&s=store.size:desc'
```

Comment:
Can you also update this sentence in the preceding paragraph?

> Then we'll pipe our output into `sort` using the appropriate column, which in this case is the eighth one.

Here's my recommendation, but please feel free to use your own:

> Then we'll sort our output by the `store.size` column, in descending order.